mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -1026,6 +1203,7 @@
         <GlossaryWeakArray value="Overseer"/>
         <GlossaryWeakArray value="Observer"/>
         <FlagArray index="UseLineOfSight" value="1"/>
+        <AttackTargetPriority value="20"/>
```

Comment:
Stasis Ward's Attack target priority increased from 10 to 20

This makes the ward not being ignored during a battle